automation/tdx_canonical_common

```diff
@@ -0,0 +1,20 @@
+CUR_DIR=$(pwd)
+GUEST_TOOLS_DIR="$TDX_DIR"/guest-tools
+GUEST_IMG_DIR="$GUEST_TOOLS_DIR"/image
+ATTESTATION_DIR="$TDX_DIR"/attestation
+BRANCH_NAME=${BRANCH_NAME:-2.0}
+is_prod_sys=1

```

Comment:
Should we set default to 0 and if its production then set to 1? Like this: https://github.com/canonical/tdx/blob/noble-24.04/setup-tdx-host.sh#L101C10-L101C31

Reply:
removed [tdx_canonical_common]